tsdb/index/index_test.go

```diff
@@ -217,7 +217,7 @@ func TestIndexRW_Postings(t *testing.T) {
 	}
 	require.NoError(t, p.Err())
 
-	// The label incides are no longer used, so test them by hand here.
+	// The label insides are no longer used, so test them by hand here.
```

Comment:
```suggestion
	// The labels inside are no longer used, so test them by hand here.
```

I think that would be better.